tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -51,6 +51,10 @@ arp/test_wr_arp.py:
     conditions:
       - "'dualtor' in topo_name"
       - https://github.com/sonic-net/sonic-buildimage/issues/16502
+  xfail:
+    reason: "Warm reboot has a known issue on 7050cx3"
+    conditions:
+      - "hwsku in ['Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8'] and release in ['202305', 'master']"
```

Comment:
@Gfrom2016 Do we need to add github issue here? If the issue is fixed, the case will run automatically.

Reply:
Raise a issue for this case: https://github.com/sonic-net/sonic-mgmt/issues/10077